src/azure-cli/azure/cli/command_modules/cosmosdb/custom.py

```diff
@@ -187,7 +199,9 @@ def cli_cosmosdb_update(client,
                         network_acl_bypass_resource_ids=None,
                         server_version=None,
                         backup_interval=None,
-                        backup_retention=None):
+                        backup_retention=None,
+                        enable_system_assigned_identity=None,
```

Comment:
Where does `enable_system_assigned_identity` come from? Do you mean `assign_identity` just like `create` operation?